src/pages/signin/LoginForm/BaseLoginForm.js

```diff
@@ -277,13 +280,23 @@ function LoginForm(props) {
                     id="username"
                     name="username"
                     testID="username"
-                    onBlur={() => {
-                        if (firstBlurred.current || !Visibility.isVisible() || !Visibility.hasFocus()) {
-                            return;
-                        }
-                        firstBlurred.current = true;
-                        validate(login);
-                    }}
+                    onBlur={
+                        // As we have only two signin buttons (Apple/Google) other than the text input,
+                        // for natives onBlur is called only when the buttons are pressed and we don't need
+                        // to validate in those case as the user has opted for other signin flow.
+                        willBlurTextInputOnTapOutside
```

Comment:
✋ Coming from https://github.com/Expensify/App/issues/42054

`willBlurTextInputOnTapOutside` is always `false` on native, leading to a bug in which the input is not validating when tapping away.